### ISSUE TITLE
Exception with dependencies from platform

### ISSUE THREAD

**Issue by stemas** [koral--/gradle-pitest-plugin#100]
Hello,

there seems to be a regression in 0.2.13 when using dependencies from a platform:
```
Execution failed for task ':app:pitestDebug'.
> Could not resolve all files for configuration ':app:debugUnitTestRuntimeClasspathCopy'.
```

This does not happen with 0.2.12.

It can be reproduces in a simple Android app project by using a platform and a dependency without version.
```
// settings.gradle.kts
include(":platform")

// platform/build.gradle.kts
plugins {
    id("java-platform")
}

dependencies {
    constraints {
        api("commons-httpclient:commons-httpclient:3.1")
    }
}

// app/build.gradle.kts
dependencies {
    testImplementation(platform(project(":platform")))
    testImplementation("commons-httpclient:commons-httpclient")
}
```

**Comment by koral--** [comment]
Thanks for the report, I'll check this.

**Comment by koral--** [comment]
Fix will be released in 0.2.14.
